lib/tests/suites/argonaut_sprint_1_test.rb

```diff
@@ -65,13 +70,13 @@ def define_metadata(method)
           validates resource: "Patient", methods: ["read", "search"]
         }
 
-        @id = @client.read_feed(@rc).resource.entry.first.resource.xmlId
+        @patient_id ||= @client.read_feed(@rc).resource.entry.first.resource.xmlId
```

Comment:
Probably. It already blows up (we get an error), so I'll add an assert or something to turn it into a fail.